controllers/storagecluster/cephcluster.go

```diff
@@ -55,6 +55,10 @@ const (
 	poolTypeMetadata = "metadata"
 )
 
+const (
+	desiredCephxKeyGenEnvVarName = "DESIRED_CEPH_X_KEY_GEN"
```

Comment:
Just a nit pick about naming conventions: `X` is either part of the Ceph X or its own word. So you need to change to one of the followings:
```suggestion
	desiredCephXKeyGenEnvVarName = "DESIRED_CEPH_X_KEY_GEN"
```
or
```suggestion
	desiredCephxKeyGenEnvVarName = "DESIRED_CEPHX_KEY_GEN"
```

I prefer the first but will accept either